Makefile

```diff
@@ -42,6 +42,12 @@ lint:
 	@echo Linting
 	@golangci-lint run --no-config --issues-exit-code=0 --timeout=5m
 
+## Format docs
+docs:
+	@echo Formatting docs
+	@npm list -g markdown-toc > /dev/null 2>&1 || npm install -g markdown-toc > /dev/null 2>&1
```

Comment:
what is this for?

Reply:
updating the table of contents (and any future stuff), i had forgotten which tool i originally used and decided to add it to the makefile so that we won't forget